store/models.py

```diff
@@ -0,0 +1,146 @@
+from django.db import models
+
+from user.models import TimeStampModel
+
+class Menu(models.Model):
+    name = models.CharField(max_length=100)
+
+    def __str__(self):
+        return self.name
+
+    class Mete:
+        db_table = 'menus'
+
+class Category(models.Model):
+    name = models.CharField(max_length=100)
+    menu = models.ManyToManyField('Menu', through='MenuCategory')
+
+    def __str__(self):
+        return self.name
+
+    class Mete:
+        db_table = 'categories'
+
+class MenuCategory(models.Model):
+    menu     = models.ForeignKey('Menu', on_delete=models.CASCADE)
+    category = models.ForeignKey('Category', on_delete=models.CASCADE)
+
+    class Mete:
+        db_table = 'menu_categories'
+
+class SubCategory(models.Model):
+    name = models.CharField(max_length=100)
+
+    def __str__(self):
+        return self.name
+
+    class Meta:
+        db_table = 'sub_categories'
+
+class ProductType(models.Model):
+    name = models.CharField(max_length=100)
+
+    def __str__(self):
+        return self.name
+
+    class Meta:
+        db_table = 'product_types'
+
+class Brand(models.Model):
+    name          = models.CharField(max_length=100)
+    free_delivery = models.DecimalField(max_digits=10, decimal_places=0, default=0)
+    category      = models.ManyToManyField('Category', through='BrandCategory')
+
+    def __str__(self):
+        return self.name
+
+    class Meta:
+        db_table = 'brands'
+
+class BrandCategory(models.Model):
+    category = models.ForeignKey('Category', on_delete=models.CASCADE)
+    brand    = models.ForeignKey('Brand', on_delete=models.CASCADE)
+
+    class Meta:
+        db_table = 'brand_categories'
+
+class Gender(models.Model):
+    name = models.CharField(max_length=10)
+
+    def __str__(self):
+        return self.name
+
+    class Meta:
+        db_table = 'genders'
+
+class Product(models.Model):
+    title        = models.CharField(max_length=200)
+    market_price = models.DecimalField(max_digits=10, decimal_places=0)
+    sale_price   = models.DecimalField(max_digits=10, decimal_places=0)
+    description  = models.TextField()
+    brand        = models.ForeignKey('Brand', on_delete=models.CASCADE)
+    product_type = models.ForeignKey('ProductType', on_delete=models.CASCADE)
+    gender       = models.ForeignKey('Gender', on_delete=models.SET_NULL, null=True)
+
+    def __str__(self):
+        return self.title
+
+    class Meta:
+        db_table = 'products'
+
+class ProductImage(models.Model):
+    image_url = models.URLField(max_length=1000)
+    product   = models.ForeignKey('Product', on_delete=models.CASCADE)
+
+    def __str__(self):
+        return self.image_url
+
+    class Meta:
+        db_table = 'product_images'
+
+class Enquiry(TimeStampModel):
+    email   = models.EmailField()
+    content = models.TextField()
+    secret  = models.BooleanField()
```

Comment:
```suggestion
    is_secret  = models.BooleanField()
```
혹은 `is_private` 도 괜찮은 이름인 것 같네요.
`boolean` field를 선언하실 때에는 `is` 또는 `has`를 붙이시는 컨벤션을 지키는 습관을 들이시면 좋습니다.

Reply:
수정했습니다